release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -264,7 +282,82 @@ For more information, see xref:../networking/dns-operator.adoc#nw-dns-cache-tuni
+[id="ocp-4-12-storage-google-cloud-file-csi-driver"]
+==== Persistent storage using the Google Filestore Driver Operator (Technology Preview)
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Google Filestore. The Google Filestore CSI Driver Operator that manages this driver is in Technology Preview.
+
+For more information, see ???.
+
+[id="ocp-4-12-storage-aws-ebs-auto-migration-ga"]
+==== Automatic CSI migration for AWS Elastic Block Storage auto migration is generally available
+Starting with {product-title} 4.8, automatic migration for in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. Support for Amazon Web Services (AWS) Elastic Block Storage (EBS) was provided in this feature in {product-title} 4.8, and {product-title} 4.12 now supports automatic migration for AWS EBS as generally available. CSI migration for AWS EBS is now enabled by default and requires no action by an administrator.
+
+This feature automatically translates in-tree objects to their counterpart CSI representations and should be completely transparent to users. Translated objects are not stored on disk, and user data is not migrated.
+
+While storage class referencing to the in-tree storage plug-in will continue working, it is recommended that you switch the default storage class to the CSI storage class.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc[CSI Automatic Migration].
+
+[id="ocp-4-12-storage-gce-auto-migration-ga"]
+==== Automatic CSI migration for GCE auto migration is generally available
+Starting with {product-title} 4.8, automatic migration for in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. Support for Google Compute Persistent Disk (GCE PD) in-tree plug-in to Google Cloud Platform (GCP) Persistent Disk CSI driver was provided in this feature in {product-title} 4.9, and {product-title} 4.12 now supports automatic migration for GCE PD to GCP Persistent Disk as generally available. CSI migration from GCE PD to GCP Persistent Disk is now enabled by default and requires no action by an administrator.
+
+This feature automatically translates in-tree objects to their counterpart CSI representations and should be completely transparent to users. Translated objects are not stored on disk, and user data is not migrated.
+
+While storage class referencing to the in-tree storage plug-in will continue working, it is recommended that you switch the default storage class to the CSI storage class.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc[CSI Automatic Migration].
+
+[id="ocp-4-12-storage-csi-inline-ephemeral-vols-ga"]
+==== CSI inline ephemeral volumes is generally available
```

Comment:
Not the case anymore needs to be removed - @lpettyjo @dobsonj

Reply:
Done